assets/data/schema.pbtxt

```diff
@@ -0,0 +1,26 @@
+feature {
+  name: "title"
+  type: BYTES
+  presence {
+    min_fraction: 1.0
+    min_count: 1
+  }
+  shape {
+    dim {
+      size: 1
+    }
+  }
+}
+feature {
+  name: "tags"
+  type: BYTES
```

Comment:
Same here

Reply:
Same thing as before, this is the schema inferred by TFDV.